README.md

```diff
@@ -120,7 +120,7 @@ $ stx make_keychain -t | jq . > cli_keychain.json
 1. Get some testnet STX:
 
 ```
-$ curl -s -X POST "localhost:3999/extended/v1/faucets/stx?address=$(cat ./cli_keychain.json | jq -r .keyInfo.address)" | jq .
+$ curl -s -X POST "https://stacks-node-api.regtest.stacks.co/extended/v1/faucets/stx?address=$(cat ./cli_keychain.json | jq -r .keyInfo.address)" | jq .
```

Comment:
`localhost` here is required to be used since this network doesn't sync with the external testnet. i.e. requesting stx from the external faucet won't be available on a private-testnet.